src/App/integrationTests/managementRegime/App.managementRegimeCreateOffline.test.js

```diff
@@ -159,9 +162,7 @@ describe('Offline', () => {
     expect(await screen.findByTestId('management-regime-toast-error')).toHaveTextContent(
       'The management regime failed to save both on your computer and in the MERMAID online system.',
     )
-    expect(await screen.findByTestId('management-regime-toast-error')).toHaveTextContent(
-      'this is a dexie error',
-    )
```

Comment:
removing an overwhelm of multiple toasts (yes this kind of is another ticket, but its related). Showing the above assertion (ln 162) is sufficient for a user who doesnt care about dexie or code details.